src/redux/reducers/name.ts

```diff
@@ -0,0 +1,10 @@
+import { SearchNameType, SEARCH_NAME } from '../types'
+
+export function nameReducer (state = '', action: SearchNameType) {
```

Comment:
what's the use of this reducer?

Reply:
Was planning to turn this into thunk that would make API call.